src/main/scala/beam/agentsim/agents/choice/logit/MultinomialLogit.scala

```diff
@@ -181,4 +181,10 @@ object MultinomialLogit {
   ): MultinomialLogit[A, T] = {
     new MultinomialLogit(utilityFunctions, commonUtilityFunction)
   }
+
+  // cannot add this constructor, because after type erasure, it looks the same as the first one. :-(
```

Comment:
Can this be removed? It serves no purpose beyond what the compiler would already tell another dev

Reply:
yep